testdata/prometheus/prometheus.base.yml

```diff
@@ -0,0 +1,37 @@
+global:
+  scrape_interval: 9m
+  scrape_timeout: 19s
+  evaluation_interval: 9m
+rule_files:
+  - /srv/prometheus/rules/test.rules.yml
+scrape_configs:
+  - job_name: mysqld_exporter_agent_id_75bb30d3-ef4a-4147-97a8-621a996611dd_hr-5s
```

Comment:
I think we should use a better test example of something that is not managed by PMM.

Reply:
Done